backend/testfiles/execution/stdlib/parser.dark

```diff
@@ -386,6 +386,9 @@ module TextToTextRoundtripping =
     // TODO: this is ugly
     ("let x = 1L\n  x" |> roundtripCliScript) = "let x =\n  1L\nx"
 
+    // field access
```

Comment:
let's also add some others:
- `(Person { name = "Janice" }).name` (just to make sure it works when there's a ()-defined record we're accessing)
- `record.someField.anotherFieldInsideThat` (nested record access)